tests/test_rio_merge.py

```diff
@@ -89,3 +89,50 @@ def test_format_jpeg(tmpdir):
         ['tests/data/RGB.byte.tif', outputname, '--format', 'JPEG'])
     assert result.exit_code == 0
     assert os.path.exists(outputname)
+
+
+# Non-coincident datasets test fixture.
+# Two overlapping GeoTIFFs, one to the NW and one to the SE.
+@fixture(scope='function')
```

Comment:
@sgillies I've been reconsidering how we are using fixtures here and in #241 based on my initial addition of them.  Obviously I didn't come up with a good precedent; the goal for fixtures should be something we reuse across multiple tests (my example originally did but I later modified the tests and ended up using it only once; I should have stripped it back out).  If the fixture is only used in one test, we could accomplish the same thing by having at the beginning of the test.  That said, your changes here are fine to leave as is for now.

I'd like to take a pass at refactoring this pattern once this and #241 are merged into the codebase.  I'm also thinking that we'll probably want to move these fixtures from this module into ```tests/conftest.py``` so we can use them in other tests (there's other tests that could benefit from using these rather than the test files for performance reasons).